sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1118,6 +1103,12 @@ public partial class ServerCertificateCustomValidationArgs
 }
 namespace Azure.Core.Serialization
 {
+    public enum DateTimeHandling
+    {
+        Iso8601 = 0,
+        Rfc3339 = 1,
```

Comment:
Do we need both `Iso8601` and `Rfc3339`?

Reply:
Iso8601 uses JsonSerializer defaults, and so is equivalent to "None" (i.e. don't add a JsonConverter for DateTime or DateTimeOffste to the serializer options).  Rfc3339 implements consistency with Azure generated models.

Reply:
Per [arch board feedback](https://github.com/Azure/azure-sdk-for-net/issues/36642), we will change to use a format string for date/time instead of the enum.